internal/dataplane/parser/translate_kong_l4_test.go

```diff
@@ -208,3 +213,239 @@ func TestFromTCPIngressV1beta1(t *testing.T) {
 		assert.Equal(2, len(parsedInfo.SecretNameToSNIs.Hosts("default/sooper-secret2")))
 	})
 }
+
+func TestIngressRulesFromTCPIngressV1beta1UsingExpressionRoutes(t *testing.T) {
+	testCases := []struct {
+		name                 string
+		tcpIngresses         []*configurationv1beta1.TCPIngress
```

Comment:
`kongv1beta1` should be used here, and in the tests below.

Reply:
Thanks, the test file hasn't been updated in time.